cabot/statuscheckapp/models/base.py

```diff
@@ -0,0 +1,137 @@
+from celery.utils.log import get_task_logger
+from django.db import models
+import json
+import time
+from cabot.cabotapp.models import CHECK_TYPES, StatusCheck, StatusCheckResult
+
+logger = get_task_logger(__name__)
+
+
+class MetricsInstanceBase(models.Model):
+    class Meta:
+        app_label = 'statuscheckapp'
+
+    name = models.CharField(
+        unique=True,
+        max_length=30,
+        help_text='Unique name for the data source',
+    )
+
+
+class MetricsStatusCheckBase(StatusCheck):
+    class Meta:
+        app_label = 'statuscheckapp'
+
+    metrics_instance = models.ForeignKey(MetricsInstanceBase)
```

Comment:
shouldn't we have `active`, `frequency` and `retries` as part of the base class?